### PR TITLE
Create ship

### PR DESCRIPTION
**Pull request by Kollurupranathi**
Implemented createShip()

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -71,7 +78,15 @@ def emptyGrid(rows, cols):
 Returns: 2D list of ints
 '''
 def createShip():
-    return
+    row = random.randint(1,8)
+    col =random.randint(1,8)
+    center =random.randint(0,1)
+    ship =[[],[],[]]
```
Comment (Pragya28) [inline battleship.py:84]: No need to initialize ship in this way. Remove this line.

**Review by Pragya28 — CHANGES_REQUESTED** [review]
Make the suggested change.
Good work @Kollurupranathi.

File: battleship.py (hunk 1 above)
Comment (Kollurupranathi) [reply battleship.py]: Resolved the issue

**Review by Pragya28 — COMMENTED** [review]
Good work @Kollurupranathi.

**Pull request merged**